jaeger-spark-dependencies-cassandra/src/main/java/io/jaegertracing/spark/dependencies/cassandra/CassandraDependenciesJob.java

```diff
@@ -173,10 +174,19 @@ public void run() {
   }
 
   private void store(JavaSparkContext sc, List<Dependency> links) {
-    CassandraDependencies dependencies = new CassandraDependencies(links, day);
-    javaFunctions(sc.parallelize(Collections.singletonList(dependencies)))
-        .writerBuilder(keyspace, "dependencies", mapToRow(CassandraDependencies.class))
-        .saveToCassandra();
+    String table = dependenciesTable(sc);
+    log.info("Storing dependencies into {}", table);
+    if (table == "dependencies_v2") {
```

Comment:
I think using equals() is better.
use == to compare strings, in this case, it works but makes confusing.